x-pack/test/api_integration/apis/beats/enroll_beat.js

```diff
@@ -97,8 +97,9 @@ export default function ({ getService }) {
         .expect(400);
 
       expect(apiResponse).to.eql({
-        success: false,
-        error: { code: 400, message: 'Invalid enrollment token' },
+        statusCode: 400,
+        error: 'Bad Request',
+        message: 'Invalid enrollment token',
```

Comment:
The legacy plugin was using custom body payload for error responses, which is not possible with KP. The client-side code actually don't care about this format changes, as the body of an error response is never accessed.